tables/SV-SE-G1.utb

```diff
@@ -0,0 +1,506 @@
+punctuation	\x201E 56		# 8222			smart double low quotation mark
+punctuation	\x201F 56		# 8223			smart double high reverse quotation mark
+punctuation \x2026 3-3-3 # 8230			smart ellipsis
+
+# Special signs
+sign # 45-3456				number sign					x0023
+sign € 45-15
+sign $ 45-234					dollar							x0024
+sign % 1456					percent							x0025
+sign ‰ 1456-1456	promille
+sign & 12346					ampersand						x0026
+sign * 35							asterisk						x002A
+noback sign \x2606 35	white star
+sign _ 45-36						underline							x005F
+sign @ 45-12356				commercial at				x0040
+sign ^ 4						circumflex accent			x005E
+sign \\ 45-34				reverse solidus				x005C
+sign | 456					vertical line					x007C
+sign ¢ 45-14					cent sign							x00A2
+sign £ 45-123				pound sign						x00A3
+sign ¤ 45-15				currency sign					x00A4
+sign ¥ 45-13456			yen										x00A5
+sign ○ 12456-1245	degrees
+sign ∙	12456-3	multiplication
+sign § 346					paragraph							x00A7
+sign © 236-14-356		copyright		x00A9
+sign ® 236-1235-356	registered trademark
+sign ™ 236-2345-134-356	trademark
+sign ° 4-356					degree sign									x00B0
+sign ² 4-6-126				superscript 2								x00B2
+sign ³ 4-6-146				superscript 3								x00B3
+sign ¹ 4-6-16					superscript 1								x00B9
+sign \x20AC 15-136-1235-135					Euro sign
+noback sign \x25CF 35	# 9679 black circle
+noback sign † 45-256	cross
+
+# Math
+math + 256						plus								x002B
+math / 34							solidus							x002F
+math < 246-3					less-than sign			x003C
+math = 2356						equal sign					x003D
+math > 135-2					greater-than sign		x003E
+math ~ 45-25					tilde									x007E
+math ¼ 3456-1-34-3456-145		vulgar fraction 1 quarter		x00BC
+math ½ 3456-1-34-3456-12		vulgar fraction one half		x00BD
+math ¾ 3456-14-34-3456-145	vulgar fraction 3 quarters	x00BE
+math × 12456-1346						multiplication sign					x00D7
+math ÷ 34							division sign								x00F7
+
+# Letters
+include latinLetterDef6Dots.uti
+include digits6Dots.uti # must come after letters
+include litdigits6Dots.uti # Must come after letters
+base uppercase \x00C0 \x00E0 
+lowercase \x00E0 12356			A with grave				x00C0
+base uppercase Ä ä
+lowercase ä 345							a with diaeresis		x00C4
+base uppercase Å å
+lowercase å 16										A with ring above		x00C5
+base uppercase \x00C8 \x00E8
+lowercase \x00E8 2346			E with grave above	x00C8
+base uppercase É é
+lowercase é 123456								E with acute above	x00C9
+base uppercase Ö ö
+lowercase ö 246																			x00D6
+base uppercase Ü ü
+lowercase ü 1256						U with diaeresis		x00DC
+
+# Capital and number signs
+capsletter 6
+begcapsword 6-6			# uncomment if you don't want capitalization
+endcapsword 156
+lencapsphrase 3
+begcapsphrase 6-6-6
+endcapsphrase after 156
+sign \x00B7 3456			# so the line below compile
+numsign 3456
+
+# Repetitions
+repeated ... 3-3-3		ellipsis
+repeated --- 36-36-36
+repeated ___ 45-36-45-36-45-36
+always \s--\s 36-36	tiret
+
+midendnumericmodechars .,\x00a0
+midnum \x00a0 3
+#numericnocontchars abcdefghij

```

Comment:
I don't like commented out rules in tables so maybe you can add some explanation, e.g. something like: "the following lines could be replaced by a single numericnocontchars rule but it does not work because ...".